db/data/20191008102825_copy_full_names_to_banking_names.rb

```diff
@@ -0,0 +1,13 @@
+class CopyFullNamesToBankingNames < ActiveRecord::Migration[5.2]
+  def up
+    Claim.where(banking_name: nil).each do |c|
+      c.update!(banking_name: c.full_name)
+    end
+  end
+
+  def down
+    Claim.all.each do |c|
+      c.banking_name = nil
```

Comment:
Do we need to commit this change as an update or save? Maybe even `Claim.update_all(banking_name: nil)`?

Reply:
Oh good catch, I'm not actually doing anything here am I?